server/src/main/java/org/elasticsearch/TransportVersion.java

```diff
@@ -460,7 +470,9 @@ private static class VersionsHolder {
             ALL_VERSIONS = Collections.unmodifiableList(allVersions);
             ALL_VERSIONS_BY_ID = ALL_VERSIONS.stream().collect(Collectors.toUnmodifiableMap(TransportVersion::id, Function.identity()));
             ALL_VERSIONS_BY_NAME = Collections.unmodifiableMap(allVersionsByName);
+
             CURRENT = ALL_VERSIONS.getLast();
+            MINIMUM_CCS_VERSION = ALL_VERSIONS_BY_ID.get(9112006);
```

Comment:
can you add a comment here that this is temporary? we need to move this to a resource file, but we'll need a new concept